Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -0,0 +1,309 @@
+            {
+                AZ_Error("CollisionMaker", false, "Error loading collider. Invalid node views.");
+                return;
+            }
+
+            // Select all nodes for both visual and collision nodes
+            for (AZ::SceneAPI::DataTypes::ISceneNodeGroup& mg : view)
+            {
+                AZ::SceneAPI::Utilities::SceneGraphSelector::SelectAll(scene->GetGraph(), mg.GetSceneNodeSelectionList());
+            }
+
+            // Update scene with all nodes selected
+            AZ::SceneAPI::Events::ProcessingResultCombiner result;
+            AZ::SceneAPI::Events::AssetImportRequestBus::BroadcastResult(
+                result,
+                &AZ::SceneAPI::Events::AssetImportRequest::UpdateManifest,
+                *scene,
+                AZ::SceneAPI::Events::AssetImportRequest::ManifestAction::Update,
+                AZ::SceneAPI::Events::AssetImportRequest::RequestingApplication::Editor);
+
+            if (result.GetResult() != AZ::SceneAPI::Events::ProcessingResult::Success)
+            {
+                AZ_Error("CollisionMaker", false, "Scene updated");
+                return;
+            }
+
+            auto assetInfoFilePath = azMeshPath;
+            assetInfoFilePath += ".assetinfo";
+            AZ_Printf("CollisionMaker", "Saving collider manifest to %s", assetInfoFilePath.c_str());
+            scene->GetManifest().SaveToFile(assetInfoFilePath.c_str());
+
+            bool assetFound = false;
+            AZ::Data::AssetInfo assetInfo;
+            AZStd::string watchDir;
+            AzToolsFramework::AssetSystemRequestBus::BroadcastResult(
+                assetFound,
+                &AzToolsFramework::AssetSystem::AssetSystemRequest::GetSourceInfoBySourcePath,
+                azMeshPath.c_str(),
+                assetInfo,
+                watchDir);
+
+            // Set export method to convex mesh
+            auto readOutcome = AZ::JsonSerializationUtils::ReadJsonFile(assetInfoFilePath.c_str());
+            if (!readOutcome.IsSuccess())
+            {
+                AZ_Error("CollisionMaker", false, "Could not read %s", assetInfoFilePath.c_str());
+                return;
+            }
+            rapidjson::Document assetInfoJson = readOutcome.TakeValue();
+            auto manifestObject = assetInfoJson.GetObject();
+            auto valuesIterator = manifestObject.FindMember("values");
+            if (valuesIterator == manifestObject.MemberEnd())
+            {
+                AZ_Error("CollisionMaker", false, "Invalid json file: %s", assetInfoFilePath.c_str());
+                return;
+            }
+            auto valuesArray = valuesIterator->value.GetArray();
+            for (auto& value : valuesArray)
+            {
+                auto object = value.GetObject();
+
+                auto physXMeshGroupIterator = object.FindMember("$type");
+                AZStd::basic_string<char> physXMeshGroupType = "{5B03C8E6-8CEE-4DA0-A7FA-CD88689DD45B} MeshGroup";
```

Comment:
Why not just use an `AZStd::string` here?
```suggestion
                AZStd::string physXMeshGroupType = "{5B03C8E6-8CEE-4DA0-A7FA-CD88689DD45B} MeshGroup";
```

Reply:
@pijaro I am correcting this, let me know if there was a specific reason